src/fr/modeles-de-page/basic/basic.md

```diff
@@ -0,0 +1,83 @@
+  descriptionSecondaryText:
+  state: published
+#  tag: Canada.ca required
+  order: 1
+translationKey: 'basicPageTemplate'
+tags: ["templates"]
+date: "git Last Modified"
+github: https://github.com/cds-snc/gcds-examples/blob/feat/add-basic-page-templates/templates/english/basic-page-template.html
+figma: https://figma.com
+---
+
+# Modèle de page de base
+
+Le modèle de page de base offre l'essentiel pour structurer une page et hiérarchiser des titres de façon accessible. Les éléments de marque, comme l'en-tête et le pied de page, permettent de reconnaître que la page vient du gouvernement du Canada.
+
+## Quand l'utiliser
+
+À utiliser sur Canada.ca et sur d'autres sites du GC.
+
+Le modèle de page de base est destiné aux pages non spécialisées de Canada.ca. Ce modèle est facultatif. Il contient les éléments de départ nécessaires pour créer une page simple et les éléments obligatoires pour la plupart des pages du GC. Vous pouvez l'utiliser comme canevas ou ajouter et agencer des <gcds-link href="{{ links.components }}">composants</gcds-link> de Système de design GC.
+
+## Aperçu
+
+<gcds-button class="mb-400" button-role="secondary" type="link" href="{{ links.pageTemplatesBasicPreview }}" target="_blank">Ouvrir demo dans un nouvel onglet</gcds-button>
+<gcds-button  button-role="secondary" type="link" href="{{ links.pageTemplatesBasicCode }}" target="_blank">Obtenez code dans un nouvel onglet</gcds-button>
+
+<img class="max-width-content b-sm b-default mb-400 p-400" src="/images/fr/templates/basic-page-preview.png" alt="Le modèle de page de base comprend l'en-tête et le pied de page du Gouvernement du Canada, ainsi que les sections titre H1, H2 et H3 accompagnées de texte sous chacune des sections."/>
+
+## Comment procéder à la mise en œuvre
+
+Pour utiliser un modèle de page dans votre projet ou réaliser un prototype en HTML, téléchargez le modèle dont vous avez besoin dans GitHub :
+
+- <gcds-link external href="{{ links.pageTemplatesBasicGithubLink }}">Modèle de page de base</gcds-link>
+- <gcds-link external href="{{ links.pageTemplatesBasicExtOTPGithubLink }}">Modèle de table des matières pour page de base</gcds-link>
+
+Pour réaliser un prototype dans Figma, trouvez le <gcds-link external href="{{ links.pageTemplatesBasicFigma }}">modèle de page de base</gcds-link> dans la bibliothèque Système de design GC.
+
+## Considérations de conception
+
+### Améliorer la navigation dans les longues pages
+
+Pour toute page ayant 4 sections ou plus, ajoutez une section « Sur cette page » comprenant une liste de liens (ancres) renvoyant aux autres sections. Les ancres améliorent la navigation et aident les gens à trouver le contenu dont ils ont besoin.
+
+### Aperçu « Sur cette page »
+
+<gcds-button class="mb-400" button-role="secondary" type="link" href="{{ links.pageTemplatesBasicExtOTPPreview }}" target="_blank">Ouvrir demo dans un nouvel onglet</gcds-button>
```

Comment:
`mb-400` needs to be changed to 300 please to match the new spacing classes.

Reply:
```suggestion
<gcds-button class="mb-300" button-role="secondary" type="link" href="{{ links.pageTemplatesBasicExtOTPPreview }}" target="_blank">Ouvrir demo dans un nouvel onglet</gcds-button>
```

Reply:
Done!